apps/android/app/src/main/java/com/axon/app/ui/settings/SettingsScreen.kt

```
@@ -52,145 +44,152 @@ import androidx.lifecycle.viewmodel.compose.viewModel
 
 private enum class SettingsTab(val label: String, val shortLabel: String, val icon: ImageVector) {
-    Connection("Connection", "Conn", Icons.Rounded.Link),
-    Env("Env", ".env", Icons.Rounded.Key),
-    Config("config.toml", "TOML", Icons.Rounded.Slideshow),
+    Connection("Connection", "Connection", Icons.Rounded.Link),
+    Env("Env", "Env", Icons.Rounded.Key),
+    Config("Config", "Config", Icons.Rounded.Slideshow),
 }
 
 @Composable
 fun SettingsScreen(vm: SettingsViewModel = viewModel()) {
```

detekt (1.23.8)

[warning] The function SettingsScreen appears to be too complex based on Cognitive Complexity (complexity: 27). Defined complexity threshold for methods is set to '25'

(detekt.complexity.CognitiveComplexMethod)